tslearn/metrics/softdtw_variants.py

```diff
@@ -308,12 +310,14 @@ def gamma_soft_dtw(dataset, n_samples=100, random_state=None):
     ----------
     .. [1] M. Cuturi, "Fast global alignment kernels," ICML 2011.
     """
-    return 2. * sigma_gak(dataset=dataset,
```

Comment:
Here I find the original version easier to read...

Reply:
Hello @rtavenar, 
Are you talking about the code aesthetics of the file softdtw_variants.py?
I think that I only did a few useful modifications in this file, most of the "aesthetics" modifications have been performed by the "black" command. The command: "black + filename" in the terminal automatically corrects Python codes aesthetics to make sure that it corresponds to PEP8 conventions. It is very practical to make sure that the PEP8 conventions are respected, but sometimes it might not be the "aesthetics" version prefered by the users. 
Would you like make to come back to the previous presentation?
And more generally, want do you think of using the "black" command? 
Personally, I find "black" very convenient, but I wouldn't want it to come at the expense of aesthetics or code clarity.

Reply:
I just think that, in this case, black made things worse instead of better-looking.

Reply:
Ok, I will restore the previous formulation.